src/libraries/Microsoft.PowerFx.Interpreter/RecalcEngine.cs

```diff
@@ -85,6 +85,14 @@ protected override IExpression CreateEvaluator(CheckResult result)
             return new ParsedExpression(irnode, ruleScopeSymbol);
         }
 
+        public static IExpression CreateEvaluator2(CheckResult result)
```

Comment:
>CreateEvaluator2 [](http://example.com/codeflow?start=34&length=16)

Ignore... didn't mean to check in. 
However , this does demonstrate that we can statically create an interpreter over a CheckResult. We don't actually need the recalc engine instance - all the state is captured in the IR by now.